curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-the-date-object-by-building-a-date-formatter/646b050091d91e036d4135a4.md

```diff
@@ -0,0 +1,74 @@
+---
+id: 646b050091d91e036d4135a4
+title: Step 12
+challengeType: 0
+dashedName: step-12
+---
+
+# --description--
+
+target the class named `title` and set the `font-size` property to `2.5rem`
```

Comment:
Let's remove this step since this is the fourth project in the curriculum. 

By this point, we are supplying the campers with all of the HTML and CSS.

Reply:
removed